site/components/PodcastCard/index.tsx

```diff
@@ -1,5 +1,9 @@
-import { Text } from "@klimadao/lib/components";
+import Image from "next/legacy/image";
```

Comment:
we can use the new Next image from now on. legacy ones will need to be migrated so might as well start using the new one!

Reply:
Done, updated to the latest on both the Card & PodcastCard.